### DISCUSSION TITLE
Your Ideas for FlashBrowser !

### DISCUSSION THREAD

**Discussion by radubirsan** [radubirsan/FlashBrowser#8] [Ideas]
How about mobile browser
How about integrated flash editor 
How about internal add network
Now about hackaton

**Comment by yoasty** [comment]
accept a url as a parameter and open with that site.

**Comment by tendajidevelopscrap** [comment]
https://archive.org/details/tradewinds-2-flash-game
https://archive.org/details/glyph-flash-game
I have a Mac, and emulators are not only hard to come by, they are hard to use. Please add these games so I can play them again.

**Reply by radubirsan** [reply]
Unfortunately I cannot find this games.

**Comment by tendajidevelopscrap** [comment]
Add the flash page as permanent browser... I open it and it only opens google now. Tried redownloaded, tried restarting. Not working..

**Reply by radubirsan** [reply]
I don't understand. FlashBrowser is not meant to be your permanent browser. You should use it to access flash content on trusted websites.

**Reply by Simo-99** [reply]
i found out that if you have a link in your clipboard it will use it as homepage. For me to set a desidered homepage i just clicked the favourite button and home button after the browser loads that page. Hope this could help you!
![Senza titolo](https://user-images.githubusercontent.com/8333352/153711948-d7ade6ca-b604-4ac4-bdb6-fe610b3bd035.png)

**Reply by tendajidevelopscrap** [reply]
ohhhh thanks so much!

**Comment by Chrysalis-Army** [comment]
Download flash files
